core/src/main/scala/kafka/server/ReplicaFetcherThread.scala

```diff
@@ -72,7 +72,7 @@ class ReplicaFetcherThread(name: String,
       metrics,
       time,
       "replica-fetcher",
-      Map("broker-id" -> sourceBroker.id.toString).asJava,
+      Map("thread-id" -> name).asJava,
```

Comment:
@guozhangwang it's nicer for users if they have have tags that are meaningful instead of having to parse components from a composite name is how I understand it.